### DISCUSSION TITLE
Logout

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#352] [Idee e proposte]
Buonasera,sarebbe possibile far modo di chiudere automaticamente il registro dopo un tot di tempo?Lo chiedo perché quando usavamo LAMPSCHOOL,c'era la possibilità da parte della segreteria di impostare un limite di tempo prima del logout ,questo rappresentava una sicurezza perché molte volte i docenti ,firmando il registro ,si dimenticavano di effettuare il logout e ovviamente chiunque poteva accedervi ,così come i docenti anche tutti i componenti presenti nel registro

**Comment by trinko** [comment]
Tecnicamente queste sono impostazioni del server, non dell'applicazione. Nel senso che, impostandole dall'applicazione, non c'è nessuna garanzia che funzionino, perché prevalgono le impostazioni del server.
Comunque, la segno come richiesta.
